vitals/Main.java

```diff
@@ -1,23 +1,64 @@
 package vitals;
 
 public class Main {
-    static boolean batteryIsOk(float temperature, float soc, float chargeRate) {
-        if(temperature < 0 || temperature > 45) {
-            System.out.println("Temperature is out of range!");
-            return false;
-        } else if(soc < 20 || soc > 80) {
-            System.out.println("State of Charge is out of range!");
-            return false;
-        } else if(chargeRate > 0.8) {
-            System.out.println("Charge Rate is out of range!");
-            return false;
-        }
-        return true;
+    
+  public static boolean isTemperatureOk(float temp) {
+    if(temp<0) {
```

Comment:
similar type of code is used for charging state also. Seems like duplicate code. Here is an opportunity to optimise and reduce code...